From 2bb7bdb6af38d90177fbec1f6943d74f2e256038 Mon Sep 17 00:00:00 2001
From: "Guillaume W. Bres" <guillaume.bressaix@gmail.com>
Date: Sat, 13 Nov 2021 14:53:34 +0100
Subject: [PATCH] do not clone boost-headers-only because we already have it thanks to BR

Leurs scripts d'install setup.py veulent des sources boost/python headers only,
a un endroit bien precis.
J'ajoute les deps vers boost::python dont on peut deja disposer, et je force
le non telechargement, qu'il n'est pas possible de realiser au sein d'un BR.

Signed-off-by: Guillaume W. Bres <guillaume.bressaix@gmail.com>
---
 scipy/_lib/setup.py | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/scipy/_lib/setup.py b/scipy/_lib/setup.py
index 94faec1a9..deb5cadce 100644
--- a/scipy/_lib/setup.py
+++ b/scipy/_lib/setup.py
@@ -2,12 +2,12 @@ import os
 import pathlib
 
 
-def check_boost_submodule():
-    from scipy._lib._boost_utils import _boost_dir
-
-    if not os.path.exists(_boost_dir(ret_path=True) / 'README.md'):
-        raise RuntimeError("Missing the `boost` submodule! Run `git submodule "
-                           "update --init` to fix this.")
+#def check_boost_submodule():
+#    from scipy._lib._boost_utils import _boost_dir
+#
+#    if not os.path.exists(_boost_dir(ret_path=True) / 'README.md'):
+#        raise RuntimeError("Missing the `boost` submodule! Run `git submodule "
+#                           "update --init` to fix this.")
 
 
 def build_clib_pre_build_hook(cmd, ext):
@@ -22,7 +22,7 @@ def configuration(parent_package='',top_path=None):
     from numpy.distutils.misc_util import Configuration
     from scipy._lib._boost_utils import _boost_dir
 
-    check_boost_submodule()
+    #check_boost_submodule()
 
     config = Configuration('_lib', parent_package, top_path)
     config.add_data_files('tests/*.py')
-- 
2.20.1

